templates/3.7/README.md

```diff
@@ -0,0 +1,269 @@
+# <a name="about"></a>About
```

Comment:
README.md and README.yaml is common for RabbitMQ 3.x, so no need to duplicate it in:
`templates/3.7`,  `templates/3.8`,  `3/debian10/3.7`,    `3/debian10/3.8`
enough to have it only once in `3/`, without templating

Reply:
added doc link for new upstream in README